### PR TITLE
vsphere: Remove visible credentials

### LINKED ISSUE #488: CAA vsphere remove visible credentials
Left over log statement prints username and password

### PR DESCRIPTION
**Pull request by caavery**
Fixes #488

Signed-off-by: Cathy Avery <cavery@redhat.com>

### CODE REVIEW

File: pkg/adaptor/hypervisor/vsphere/service.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ type hypervisorService struct {
 }
 
 func newService(govmomiClient *govmomi.Client, config *Config, hypervisorConfig *hypervisor.Config, workerNode podnetwork.WorkerNode, podsDir, daemonPort string) pb.HypervisorService {
-	logger.Printf("service config %v", config)
+
```
Comment (bpradipt) [inline pkg/adaptor/hypervisor/vsphere/service.go:47]: How about using `logger.Printf("service config %v", config.Redact())` ?

Comment (caavery) [reply pkg/adaptor/hypervisor/vsphere/service.go:47]: It is an unnecessary duplicate log statement. That info is already printed out in  NewServer.

Comment (bpradipt) [reply pkg/adaptor/hypervisor/vsphere/service.go:47]: ok

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @caavery

**Review by whitebrandy — APPROVED** [review]
lgtm

**Pull request merged**